src/components/ContentSidebar/SidebarVersions.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * @flow
+ * @file Versions sidebar component
+ * @author Box
+ */
+
+import React from 'react';
+import VersionHistoryLink from 'box-react-ui/lib/features/item-details/VersionHistoryLink';
+import type { FileVersions } from '../../flowTypes';
+
+type Props = {
+    onVersionHistoryClick?: Function,
+    versions: FileVersions
+};
+
+const SidebarVersions = ({ onVersionHistoryClick, versions }: Props) => {
+    const { total_count } = versions;
+
+    if (!total_count) {
+        return null;
+    }
+
+    return (
+        <section className='bcs-section'>
+            <VersionHistoryLink onClick={onVersionHistoryClick} versionCount={total_count + 1} />
+        </section>
+    );
+};
+
+SidebarVersions.defaultProps = {
```

Comment:
Not the pattern we use, causes side effects for uglifyjs cleanup.
Assign the defaults within the props.

{ onVersionHistoryClick, versions = { total_count: 0 } }

Also will total count not be there?

Reply:
total_count would not be there if API hasn't responded by time the component has rendered